### PR TITLE
Issue 51 move variables to new topics

### PR DESCRIPTION
**Pull request by ollylucl**
Example code that shows how to reassign variables to new topics, and verify that these topic reassignments have been successful. Deals with https://github.com/CLOSER-Cohorts/colectica-api/issues/51

### CODE REVIEW

File: examples/lib/utility.py
Comment (spuddybike) [inline examples/lib/utility.py:1]: examples/update_variables.py
line 81 
INPUT_FILE_NAME = "Topics_to_be_changed.xlsx" please change to an input argument

**Review by spuddybike — COMMENTED** [review]
one comment

File: examples/lib/utility.py
Comment (ollylucl) [reply examples/lib/utility.py:1]: I've pushed up a commit that deals with this.

**Comment by ollylucl** [comment]
@spuddybike @jli755 Can I merge this PR into the master branch?

**Pull request merged**